doc/contributor/howto-guide-adding-generated-libraries.md

```diff
@@ -253,6 +253,17 @@ Announce the new library in the CHANGELOG for the next release.
 ci/cloudbuild/build.sh -t checkers-pr
 ```
 
+## Add the API baseline
+
+For new GA libraries you need to create the API baseline.
+
+```
+ci/cloudbuild/build.sh -t check-api-pr
+git add ci/abi-dumps/google_cloud_cpp_${library}.expected.abi.dump.gz
+git commit -m"Add API baseline" ci/abi-dumps/google_cloud_cpp_${library}.expected.abi.dump.gz
+git checkout -- ci/abi-dumps/
```

Comment:
Why is this needed?  Isn't the working tree already in the right shape?

Reply:
The working tree contains (trivial) changes for all API dumps.  And we are going to commit the full directory later.  I would rather avoid the churn.

Reply:
I see.  Might not `git restore ci/abi-dumps` be clearer?